script.js

```diff
@@ -0,0 +1,35 @@
+const key = "36150daea36692d65248e6c1d9a32902";
+let weatherInfo = document.getElementById("weather-info");
+let searchBtn = document.getElementById("search-btn");
+let city = document.getElementById("city");
+
+let getWeather = () => {
+  let cityValue = city.value;
+  if (cityValue.length == 0) {
+    weatherInfo.innerHTML = `<h3>Please enter a city name</h3>`;
+  } else {
+    let url = `https://api.openweathermap.org/data/2.5/weather?q=${cityValue}&appid=${key}&units=metric`;
+    fetch(url)
+      .then((response) => response.json())
+      .then((data) => {
+        console.log(data);
+        weatherInfo.innerHTML = `<h2>${data.name}</h2>`;
```

Comment:
It's not a good approach to add html tags in the innerHTML. https://medium.com/@verity.carlos/why-you-shouldnt-use-innerhtml-and-what-to-use-instead-ed99d064a416

Try creating new elements like we did in the lessons instead!